packages/suite/src/components/suite/banners/MessageSystemBanner.tsx

```diff
@@ -64,11 +55,31 @@ export const MessageSystemBanner = ({ message }: MessageSystemBannerProps) => {
     }, [id, dismissible, dispatch]);
 
     return (
-        <BannerOnTop
+        <Warning
+            icon
             variant={variant === 'critical' ? 'destructive' : variant}
-            body={content[language] || content.en}
-            action={actionConfig}
-            dismissal={dismissalConfig}
-        />
+            rightContent={
+                <Row gap={8}>
```

Comment:
I know this is handled by typescript, but I just want to make sure that we want to use it like this and not explicitly via spacingsPx?

Reply:
I will fix it in next PR